AddTransaction.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from 'vitest'
+import { mount } from '@vue/test-utils'
+import AddTransaction from '@/components/AddTransaction.vue'
+
+describe('AddTransaction elements', () => {
+  const AddTransactionWrapper = mount(AddTransaction)
+
+  it('should render correctly', () => {
```

Comment:
always double quote first.

```suggestion
  it("should render correctly", () => {
```